### DISCUSSION TITLE
Envio de Datos

### DISCUSSION THREAD

**Discussion by jricardocam** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#493] [Q&A]
Hola a todos!!
Mi duda es como enviar los datos que se llenaron en el formulario de pedido de pastel a la pagina del pastelero
gracias!

**Comment by Ingrzs** [comment]
Pues cuando llenes los datos del cliente,todos esos datos guardalos en el estado, ya luego lo recuperas en la pagina del pastelero ,  checa como hizo el mc con la funcion de elevar el número al cuadrado

**Reply by OmarVenturaP** [reply]
eso quiero hacer yo tambien. Pero lo que hizo el MC fue en la misma página no cambió a otro lado de la página, ¿aquí al cambiar de página no se pierden esos datos, digo pues como no quedan en Backend? Eso es lo que me está quebrando la cabeza también.

**Reply by ErickMontesDK** [reply]
Estoy en las mismas. Puedo mostrar los datos del formulario en la misma página, pero ni idea de como hacer que se vean esos mismos datos en la del pastelero

**Reply by jricardocam** [reply]
Logro mostrar los datos mientras se llena el formulario, pero al dar clic al boton no me guarda nigun dato

**Reply by AngelCruzO** [reply]
Yo logré pasar datos con el botón, el problema que tengo es que solo me guarda el primer input, los demás me los da undefined

**Reply by enriqueorea** [reply]
Hola, al igual que ustedes actualmente estoy aprendiendo VUE.JS sin embargo, llevo un poco más de tiempo utilizando REACT el cual al igual que VUE.JS utilizan componentes y son muy similares.

la lógica de la practica del pastelero es la siguiente:

- utilizar un form que al hacer submit cree un objeto con los datos ingresados por el usuario.
- el objeto creado se almacenara en una referencia que guarde un array de los objetos
- esa referencia sera consumida por el componente pastelero que sera enviado del componente del formulario al componente de pastelero mediante PROPS

**Reply by martinGM05** [reply]
Lo que dijo el de arriba, básicamente se tiene que conocer lo que son arreglos y objetos, ya que una orden debería ser un objeto y básicamente órdenes es el arreglo de esos objetos
